### PR TITLE
Add PCSX-ReARMed standalone in experimental section

### PR DESCRIPTION
**Pull request by schmurtzm**
PCSX-ReARMed added with enhanced resolution enabled by default.
Up to date (commit 029e681)

### CODE REVIEW

File: static/packages/RApp/Sony - PlayStation (PCSX standalone)/RApp/PCSX-ReARMed/config.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+"label":"PCSX-ReARMed",
+"icon":"pcsx_rearmed.png",
+"iconsel":"",
+"launch":"launch.sh",
+"rompath":"../../Roms/PS",
+"imgpath":"../../Imgs/PS",
+"unused_rroms_rompath":"../../RRoms/pcsx_rearmed",
```
Comment (Aemiii91) [inline static/packages/RApp/Sony - PlayStation (PCSX standalone)/RApp/PCSX-ReARMed/config.json:8]: Remove this?

File: static/packages/RApp/Sony - PlayStation (PCSX standalone)/RApp/PCSX-ReARMed/config.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+"label":"PCSX-ReARMed",
+"icon":"pcsx_rearmed.png",
+"iconsel":"",
+"launch":"launch.sh",
+"rompath":"../../Roms/PS",
+"imgpath":"../../Imgs/PS",
```
Comment (Aemiii91) [inline static/packages/RApp/Sony - PlayStation (PCSX standalone)/RApp/PCSX-ReARMed/config.json:7]: Should be `../../Roms/PS/Imgs`

File: static/packages/RApp/Sony - PlayStation (PCSX standalone)/RApp/PCSX-ReARMed/launch.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+echo $0 $*
+progdir=`dirname "$0"`
+homedir=`dirname "$1"`
+ROMNAME="$1"
+BASEROMNAME=${ROMNAME##*/}
+ROMNAMETMP=${BASEROMNAME%.*}
+
+cd $progdir
+echo test
```
Comment (Aemiii91) [inline static/packages/RApp/Sony - PlayStation (PCSX standalone)/RApp/PCSX-ReARMed/launch.sh:10]: Delete

**Review by Aemiii91 — CHANGES_REQUESTED** [review]

File: static/packages/RApp/Sony - PlayStation (PCSX standalone)/RApp/PCSX-ReARMed/launch.sh (hunk 3 above)
Comment (schmurtzm) [reply static/packages/RApp/Sony - PlayStation (PCSX standalone)/RApp/PCSX-ReARMed/launch.sh]: Done ;)

**Review by Aemiii91 — DISMISSED** [review]

**Review by Aemiii91 — APPROVED** [review]

**Pull request merged**